.github/workflows/ci.yml

```diff
@@ -328,6 +326,8 @@ jobs:
             awk 'BEGIN{FS="="} {print $3}' | sed s'/["}]//g')
           echo "DEBIAN_VERSION=${DEBIAN_VERSION}" >> $GITHUB_ENV
         if: needs.build-info.outputs.inWorkflowBuild == 'true'
+      - run: ./scripts/ci/install_breeze.sh
+        if: needs.build-info.outputs.inWorkflowBuild == 'true'
```

Comment:
The order here is all I really see different between CI and PROD, so hopefully reordering them will help?

Reply:
Not really :)

Reply:
🙃 it was an attempt.